gke-policies/policy/monitoring_and_logging.rego

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# METADATA
+# title: GKE Monitoring and Logging
```

Comment:
what do you thing about using "Cloud Monitoring and Logging" here instead of a "GKE Mon..." ?
IMHO it will be more clear what we are referring to (without a need to read description)

Reply:
good point! corrected